includes/Modules/Analytics/Proxy_Account_Ticket.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Proxy_AccountTicket
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics;
+
+use Google\Site_Kit_Dependencies\Google_Service_Analytics_AccountTicket;
+
+/**
+ * Class for the Analytics provisioning Account Ticket object.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Proxy_AccountTicket extends Google_Service_Analytics_AccountTicket {
+
+	/**
+	 * The site ID.
+	 *
+	 * @since n.e.x.t
+	 * @var String
+	 */
+	public $site_id = '';
+
+	/**
+	 * The site secret.
+	 *
+	 * @since n.e.x.t
+	 * @var String
+	 */
+	public $site_secret = '';
```

Comment:
@adamsilverstein why are these public? It seems odd when there are getters and setters for them. Is this related to how the underlying classes work?

Reply:
yes, this is because the public elements are sent when the account ticket is sent in the api request. The setters and getters follow the pattern from default `Account_Ticket` class.